src/synthesize.jl

```diff
@@ -152,7 +152,8 @@ function synthesize(atm::ModelAtmosphere, linelist, A_X::Vector{<:Real}, λs::Ab
     end
     #put number densities in a dict of vectors, rather than a vector of dicts.
     n_dicts = first.(pairs)
-    number_densities = Dict([spec=>[n[spec] for n in n_dicts] for spec in keys(n_dicts[1])])
+    number_densities = Dict([spec=>[n[spec] for n in n_dicts] for spec in keys(n_dicts[1]) 
+                             if spec != species"H III"])
```

Comment:
This may not necessarily be the time to make this change, but should the species constructor throw an error if a user tries to specify an ion that corresponds to charge that exceeds the atomic number?

Reply:
That is a good idea.  I will add it to #1.